### DISCUSSION TITLE
无法获取到机器人的配置文件

### DISCUSSION THREAD

**Discussion by ChengYull** [simple-robot/Discussions#116] [打扰了]
### 涉及的编程语言

Java

### 疑问描述

 The number of Installable event provider Factories is 0 一直显示0  但是我配置了mybot.bot.json

**Comment by ForteScarlet** [comment]
任何与预期不符、疑似bug或有异常的错误请反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml)
